docs/content/administration/config-cheat-sheet.en-us.md

```diff
@@ -755,6 +755,7 @@ and
 - `HELO_HOSTNAME`: **(retrieved from system)**: HELO hostname.
 - `FROM`: **_empty_**: Mail from address, RFC 5322. This can be just an email address, or the "Name" \<email@example.com\> format.
 - `ENVELOPE_FROM`: **_empty_**: Address set as the From address on the SMTP mail envelope. Set to `<>` to send an empty address.
+- `FROM_DISPLAY_NAME_FORMAT`: **{{ .DisplayName }}**: If gitea sends mails on behave of users, it will just use the name also displayed in the WebUI. If you want e.g. `Mister X (by CodeIt) <gitea@codeit.net>`, set it to `{{ .DisplayName }} (by {{ .AppName }})`. Beside **DisplayName** and **AppName** we have support for **Domain** too.
```

Comment:
```suggestion
- `FROM_DISPLAY_NAME_FORMAT`: **{{ .DisplayName }}**: If gitea sends mails on behave of users, it will just use the name also displayed in the WebUI. If you want e.g. `Mister X (by CodeIt) <gitea@codeit.net>`, set it to `{{ .DisplayName }} (by {{ .AppName }})`. Available Variables: `.DisplayName`, `.AppName` and `.Domain`.
```